test/pipeline_test.go

```
@@ -0,0 +1,45 @@
+
+func TestDGA_BFS(t *testing.T) {
+	dgaGraph := pipelinex.NewDGAGraph()
+	node1 := pipelinex.NewDGANode("a", "RUNNING")
+	node2 := pipelinex.NewDGANode("b", "UNKNOWN")
+	node3 := pipelinex.NewDGANode("c", "UNKNOWN	")
+	node4 := pipelinex.NewDGANode("e", "UNKNOWN")
+	dgaGraph.AddVertex(node1)
+	dgaGraph.AddVertex(node2)
+	dgaGraph.AddVertex(node3)
+	dgaGraph.AddVertex(node4)
+	dgaGraph.AddEdge(node1, node2)
```

golangci-lint (1.64.8)

Error return value of `dgaGraph.AddEdge` is not checked

(errcheck)